src/tig.c

```diff
@@ -292,6 +292,11 @@ view_driver(struct view *view, enum request request)
 		find_next(view, request);
 		break;
 
+	case REQ_MOVE_NEXT_MERGE:
+	case REQ_MOVE_PREV_MERGE:
+		find_merge(view, request);
```

Comment:
This will segfault on views that does not have a graph. I suggest this only reports an error:

```c
report("Moving between merge commits is not supported by the %s view", view->name);
```

And the `find_merge` cases are moved to `main_request` in `src/main.c`.